modules/swagger-codegen/src/main/resources/Java/libraries/resttemplate/ApiClient.mustache

```diff
@@ -268,6 +268,8 @@ public class ApiClient {
      * @return ApiClient this client
      */
     public ApiClient addDefaultHeader(String name, String value) {
+        if(defaultHeaders.containsKey(name))
```

Comment:
@JacC510 can you please add a space after `if`? (style guide: https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md#style-guide)